src/settings.cpp

```diff
@@ -901,6 +902,7 @@ void CServerSettings::ReadSettingsFromXML ( const QDomDocument& IniXMLDocument,
             {
```

Comment:
Should the `else` block at line 895 be enclosed in braces, too, for the sake of consistency?

Reply:
I wouldn't have thought so. We only need to solve the clang confusion due to the comments being between the `else` on 877 and the `if` belonging to it on 885.